cmd/kuberhealthy/main.go

```diff
@@ -92,6 +97,24 @@ func init() {
 		masterCalculation.DebugAlwaysMasterOn()
 	}
 
+	// if tolerations are passed in, format them for use in the daemon set
+	if tolerations != nil {
+		for _, t := range tolerations {
+			formattedTolerations = append(formattedTolerations, apiv1.Toleration{Key: t})
+		}
+	}
+	// if we arent passing a list of tolerations, generate our own
```

Comment:
Move this one into the daemonset checker so that it runs each time the check runs.  This probably means refactoring the `New()` and `Run()` methods in the daemonset checker package